src/components/modals/send-modal/index.js

```diff
@@ -34,7 +36,40 @@ const SendModal = ({ show, handleModal, utxo, onSale }) => {
         }
     }, []);
 
-    async function sendUtxo() {
+    async function sendUtxo(boost = false) {
```

Comment:
nit: Is this param actually necessary? Would be fairly lightweight to calculate on the fly